src/react-apps/applications/runtime/src/shared/resources/queue/dataTask/dataTaskQueueActions.ts

```diff
@@ -0,0 +1,11 @@
+import { Action } from 'redux';
+import * as ActionTypes from './dataTaskQueueActionTypes';
+
+export interface IDataTaskQueue extends Action {
+}
```

Comment:
No need to add an interface here and extend it. Just make the function return an `Action`.

Reply:
fixed.